docs/metrics-storage-management/enable-streaming.mdx

```diff
@@ -72,6 +72,45 @@ looks like the following:
 Save the file and close it, then restart Netdata with `sudo systemctl restart netdata`, or the [appropriate
 method](/docs/configure/start-stop-restart.md) for your system.
 
+## Enable TLS/SSL on streaming (optional)
+
+While encrypting the connection between your parent and child nodes is recommended for security, it's not required to
+get started. If you're not interested in encryption, skip ahead to [view streamed
+metrics](#view-streamed-metrics-in-netdata-s-dashboard).
+
+In this example, we'll use self-signed certificates. 
+
+On the **parent** node, use OpenSSL to create the key and certificate, then use `chown` to make the new files readable
+by the `netdata` user.
+
+```bash
+sudo openssl req -newkey rsa:2048 -nodes -sha512 -x509 -days 365 -keyout /etc/netdata/ssl/key.pem -out /etc/netdata/ssl/cert.pem
+sudo chown netdata:netdata /etc/netdata/ssl/cert.pem k/etc/netdata/ssl/ey.pem
+```
+
+Next, enforce TLS/SSL on the web server. Open `netdata.conf`, scroll down to the `[web]` section, and look for the `bind
+to` setting. Add `^SSL=force` to turn on TLS/SSL.
```

Comment:
I think it is a good idea to add a link here for the other options available for `^SSL=`.

Reply:
Good idea. Done!